kolibri/core/assets/src/vue/core-base.vue

```diff
@@ -60,10 +72,37 @@
       title(newVal, oldVal) {
         document.title = `${newVal} - Kolibri`;
       },
+      'windowSize.breakpoint': function (newVal, oldVal) {
```

Comment:
I did wonder if it might throw an error. Will have to make it ignore, as it can't be done as a method.